types/stylelint/index.d.ts

```diff
@@ -251,6 +252,22 @@ declare namespace stylelint {
 		fixable?: boolean;
 	};
 
+	export type Range = {
+		start: {
+			line: number;
+			column: number /** inclusive */;
+		};
+		end: {
+			line: number;
+			column: number /** exclusive */;
```

Comment:
[ask] What "inclusive" and "exclusive" in the comments mean?

Reply:
It works like `column` and `endColumn` in our tests.
i.e. `end.column` is the index of the `\0`